apps/elewa-group-website/src/app/app.module.ts

```diff
@@ -8,10 +8,12 @@ import { AppComponent } from './app.component';
 import { HomePageModule } from '@elewa-group/pages/elewa/home';
 
 import { CardsModule } from '@elewa-group/features/components/cards';
+import { ElewaGroupImageAndTextBannerComponent } from 'libs/features/components/banners/src/lib/banners/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component';
```

Comment:
Use domain imports not relative imports i.e 

import { ElewaGroupImageAndTextBannerComponent } from '@elewa-group/features/components/banners'